app.rb

```diff
@@ -30,12 +30,15 @@ class MakersBnb < Sinatra::Base
   post '/listings' do
     Listing.create(
       name: params[:name],
+      description: params[:description],
+      price: params[:price]
     )
     redirect '/listings'
   end
 
   get '/listings' do
     @listings = Listing.all
+    
```

Comment:
Can we remove this blank like to keep consistency with the rest of the codebase please?